UPGRADE-3.x.md

```diff
@@ -4,6 +4,17 @@ UPGRADE 3.x
 UPGRADE FROM 3.x to 3.x
 =======================
 
+### 'icon' option
+
+We used multiple icon option with different format:
+- `<i class="fa fa-plus"></i>`
+- `fa fa-plus`
+- `fa-plus`
+- `plus`
+
+The last two format are now deprecated.  Since we've fixed the deprecation in the code,
```

Comment:
Probably we need to update some docs, IMO we could give preference to the fa fa-icon syntax (but mentioning that the html works too)